app/src/pages/inside/projectSettingsPageContainer/content/defectTypes/defectTypeRow/defectTypeRow.jsx

```diff
@@ -155,7 +155,9 @@ export const DefectTypeRow = ({
             )}
           </div>
         </div>
-        <DefectLocatorWithPopover locator={locator} />
+        <div className={cx('locator-icon-wrapper')}>
```

Comment:
You can throw cx('locator-icon-wrapper') to withPopover as popoverWrapperClassName.